cloudinit/ssh_util.py

```diff
@@ -148,11 +148,15 @@ def parse(self, src_line, options=None):
 
         def parse_ssh_key(ent):
             # return ketype, key, [comment]
-            toks = ent.split(None, 2)
+            toks = ent.split(' ')
             if len(toks) < 2:
                 raise TypeError("To few fields: %s" % len(toks))
+            if len(toks) > 3:
```

Comment:
Given this, I'm not 100% sure how we can distinguish between accidentally-truncated-because-of-unexpected-whitespace base64 content, and a base64-encoded comment field.